docs/api.md

```diff
@@ -255,6 +255,8 @@ for `code` and `errno` are:
   This email can not currently be used to login
 * `code: 400, errno: 150`:
   Can not resend email code to an email that does not belong to this account
+* `code: 400, errno: 151`:
```

Comment:
I'm not sure why these documentation changes ended up here, I imaging the pre-push hook kicked in.

Reply:
Yep, feel free to submit them as a separate PR if you don't want them stinking up this changeset, or not if you don't mind them.